CSS/style.css

```diff
@@ -830,6 +845,148 @@ hr {
     width: 73%;
   }
 
+  .languages-list li {
+    display: flex;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    margin-right: 20px;
+  }
+
+  .languages-list ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages-list > ul > li {
+    display: flex;
+
+    /* background: #f7f7f9; */
+    border-radius: 8px;
+    margin-bottom: 12px;
+    margin-top: 21px;
+    width: 120px;
+    height: 112px;
+  }
+
+  .languages-list p {
+    font-size: 17px;
+  }
+
+  .contact {
+    display: flex;
+    width: 100%;
+    height: 920px;
+    flex-direction: column;
+    background-image: url("../img/form.png");
+    background-repeat: no-repeat;
+    background-size: inherit;
+    background-position-x: right;
+    background-color: #6070ff;
+    border-top-left-radius: 90px;
+  }
+
+  .descr {
+    display: flex;
+    flex-direction: column;
+    width: 30%;
+    padding-left: 450px;
+  }
+
+  .descr h1 {
+    width: 90%;
+    height: 56px;
+    margin-top: 100px;
+    margin-left: 20%;
+    font-size: 40px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: center;
+    color: #fff;
+  }
+
+  .descr p {
```

Comment:
Kindly update the styling of each of your cards to match the figma design better🙏

Your design:
<img width="388" alt="image" src="https://user-images.githubusercontent.com/37116322/230653657-8de619be-a375-4203-bd15-d266fb8eacee.png">

Figma design:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/37116322/230653710-9abdf6b2-8406-4f83-886f-98d323a0fbd5.png">

A few things to note:
- Kindly reduce the font size of the paragraph text
- Kindly increase the spacing between each of the elements in the card details
- Kindly increase the top margin between the title of the card and the top of the card